src/ca/noxid/lab/gameinfo/CSExe.java

```diff
@@ -150,6 +150,10 @@ else if (segStr.contains(".rdata")) //$NON-NLS-1$
 				csHead.order(ByteOrder.LITTLE_ENDIAN);
 				csHead.putInt(0x14, mapdataLoc);
 				ExeSec csmapSec = new ExeSec(csHead, chan);
+				if (headers.length - 1 != rsrcSec) {
+					StrTools.msgBox(Messages.getString("CSExe.9") + Messages.getString("CSExe.10")); //$NON-NLS-1$
```

Comment:
i'll get you to concatenate these as well as the messages for Sue's Workshop if you don't mind :)
the only reason it was done like that before is because the initial string extraction was done programmatically and it didn't have the nuance to fix situations like this.